corehq/apps/reports/filters/users.py

```diff
@@ -183,7 +183,7 @@ def id_to_choice_tuple(self, id_):
             raise Exception("Unexpcted id: {}".format(id_))
 
         if hasattr(owner, 'is_deleted'):
-            if (callable(owner.is_deleted) and owner.is_deleted()) or owner.is_deleted == True:
+            if (callable(owner.is_deleted) and owner.is_deleted()) or owner.is_deleted is True:
```

Comment:
Could we refactor this to simply be `or owner.is_deleted`?

Reply:
Maybe!

I've tried to be careful to keep code changes in these PRs to the absolute minimum (sometimes it's unavoidable when fixing bare exceptions) so that might be worth investigating in a follow-up PR.

(I'm trying to keep the risk of regressions as close to 0 as possible, and to minimize the chance of merge conflicts with anyone else's work, considering how many files can be involved when linting big areas of HQ.)

Reply:
(To be clear, I'm sure you're right! It wouldn't make sense for an `is_deleted` property or field to have a truthy value that _shouldn't_ be treated as `True`.)

Reply:
Actually, I think your suggestion is better and safer than the linted version.

```python
>>> is_deleted = 1
>>> is_deleted == True
True
>>> is_deleted is True
False
```

Reply:
a1f1e92ba97